src/components/ParserComponent.tsx

```diff
@@ -303,7 +314,7 @@ export default class ParserComponent extends React.Component<ParserInterface,
         }
       } catch (e) {
         alert('Wrong file type was uploaded.');
-        console.log('Wrong file was uploaded.');
+        console.log('Wrong file type was uploaded.');
```

Comment:
This doesn't make much sense? How does an exception here mean the wrong file type was uploaded? You seem to check the type with an if statement.